### DISCUSSION TITLE
How to trans the attributes in Liwewire?

### DISCUSSION THREAD

**Discussion by jeanali** [livewire/livewire#1722] [4. General]
My app is multi languages and I have a validation in component as follows:

<img width="483" alt="Screen Shot 2020-09-29 at 11 09 32 PM" src="https://user-images.githubusercontent.com/67483209/94610356-d9657b80-02a8-11eb-9858-a3d27a03aa86.png">

with above code I got this error:

`ErrorException
mb_strtoupper() expects parameter 1 to be string, array given 
`

How can I translate the attributes ?

**Comment by iksaku** [comment]
Make sure that your translation keys are properly configured, if that's where the error is. Otherwise, please verify your component logic.

Could you provide a [Playground Snippet](https://laravelplayground.com/#/) with your component code?
